src/util/updates.js

```diff
@@ -0,0 +1,20 @@
+/* @flow */
+import defaultUpdateNotifier from 'update-notifier';
+
+type checkForUpdatesParams = {
+  version: string,
+  updateNotifier?: typeof defaultUpdateNotifier,
+};
```

Comment:
This should use the new pipe syntax in Flow that says "only these parameters are allowed" : 
````javascript
type checkForUpdatesParams = {|
  version: string,
  updateNotifier?: typeof defaultUpdateNotifier,
|};
````